scripts/globals/abilities/stay.lua

```diff
@@ -21,6 +25,30 @@ end;
 -- onUseAbility
 -----------------------------------
 
-function onUseAbility(player,target,ability)
-    player:petStay();
+function onUseAbility(player,target,ability,action)
+    pet = player:getPet();
```

Comment:
Can you explain to me exactly what that means?  Like I said, I’m a lua noob, and I thought that *was* local.

 

Does it need to be declared like this?

local pet = player:getPet();

 

From: Tahir Akhlaq [mailto:notifications@github.com] 
Sent: Friday, March 4, 2016 3:40 PM
To: DarkstarProject/darkstar <darkstar@noreply.github.com>
Cc: lenonk <lenon@sanctuary.org>
Subject: Re: [darkstar] BST stay command now works per retail. The amount healed may need to be (#2849)

 

In scripts/globals/abilities/stay.lua <https://github.com/DarkstarProject/darkstar/pull/2849#discussion_r55092756> :

> @@ -21,6 +25,30 @@ end;
>  -- onUseAbility
>  -----------------------------------
>  
> -function onUseAbility(player,target,ability)
> -    player:petStay();
> +function onUseAbility(player,target,ability,action)
> +    pet = player:getPet();

whoops missed this, should be local

—
Reply to this email directly or view it on GitHub <https://github.com/DarkstarProject/darkstar/pull/2849/files#r55092756> .  <https://github.com/notifications/beacon/ABIIJTDnC2z_mV20XL2XejnM4lWwTfdnks5pqKabgaJpZM4HnQDO.gif>